resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -63,64 +63,105 @@ Item
         // the cloud contents area.
         Column
         {
-            anchors.centerIn: parent
+            anchors.horizontalCenter: parent.horizontalCenter
             width: parent.width
             height: childrenRect.height
 
             spacing: 20 * screenScaleFactor
 
-            Image  // Cloud image
+            // Cloud image
+            Image
             {
                 id: cloudImage
                 anchors.horizontalCenter: parent.horizontalCenter
                 source: UM.Theme.getImage("first_run_ultimaker_cloud")
+                scale: 0.666
             }
 
-            Label  // A title-ish text
+            // Motivational icons
+            Row
             {
-                id: highlightTextLabel
-                anchors.horizontalCenter: parent.horizontalCenter
-                horizontalAlignment: Text.AlignHCenter
-                text: catalog.i18nc("@text", "Your key to connected 3D printing")
-                textFormat: Text.RichText
-                color: UM.Theme.getColor("primary")
-                font: UM.Theme.getFont("medium")
-                renderType: Text.NativeRendering
-            }
+                id: motivationRow
+                width: parent.width
 
-            Label  // A number of text items
-            {
-                id: textLabel
-                anchors.horizontalCenter: parent.horizontalCenter
-                text:
+                Column
                 {
-                    // There are 3 text items, each of which is translated separately as a single piece of text.
-                    var full_text = ""
-                    var t = ""
-
-                    t = catalog.i18nc("@text", "- Customize your experience with more print profiles and plugins")
-                    full_text += "<p>" + t + "</p>"
-
-                    t = catalog.i18nc("@text", "- Stay flexible by syncing your setup and loading it anywhere")
-                    full_text += "<p>" + t + "</p>"
+                    id: marketplaceColumn
+                    width: Math.round(parent.width / 3)
+                    spacing: UM.Theme.getSize("default_margin").height
+
+                    Image
+                    {
+                        id: marketplaceImage
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        source: UM.Theme.getIcon("plugin_48px")
+                    }
+                    Label
+                    {
+                        id: marketplaceTextLabel
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        text: catalog.i18nc("@text", "Add material settings and\nplugins from the Marketplace")
```

Comment:
Rather than including a newline in the source text, it's better to put this label in an appropriately sized box and let it wrap automatically. Setting the `wrapMode` property to `Text.Wrap` should do the trick.

The newline won't be translated very cleanly. The translator doesn't know what sort of width it can take up per line.